### ISSUE TITLE
Reproducing results on Reuters Dataset

### ISSUE THREAD

**Issue by Ravikiran1408** [eelxpeng/dec-pytorch#1]
Thank you for your contribution!
I tried executing the DEC code on reuters10k dataset, and I getting 70% clustering accuracy. The authors however report 72% accuracy on the same. Apart from the input dimension and number of clusters, i have not changed any of the variables/hyperparameters. I did play around with the learning rate during pre-training, but the results did not improve. Any ideas on how to proceed? Appreciate your help.
